### DISCUSSION TITLE
Make sure you backup any firmware from tititopher68 as everything is gone

### DISCUSSION THREAD

**Discussion by Tapap84** [Jyers/Marlin#1990] [General Discussion]
I have a download from 6/18/22 if anyone needs i can email it

**Comment by darco415** [comment]
wow, so I guess Github sided w/LCH?

**Reply by brunohenriquy** [reply]
What is LCH?

**Comment by Tapap84** [comment]
Idk, tititopher probably just deleted his account but who knows. I wish I
saved the link the other day for his discord. I already switched to klipper
on 2/3 of my machines and I'll run jyers/tititopher on the third just for
giggles. I won't use any other branch of marlin.

**Reply by darco415** [reply]
this is the jstech discord where the fw is shared: https://discord.gg/VHN4mChd ,,, be warned, its in french

**Comment by Nbrooks639** [comment]
they let him back in

**Comment by Tapap84** [comment]
You have links

**Comment by ghost** [comment]
I'm so sick of the drama all around this, I don't even know what happened this time. Doesn't matter who you side with, people should just let each other be. Fucking hell.

I'm about to give up on Marlin and go with Klipper.

**Reply by FunkyNuggets007** [reply]
I did klipper on my S1 Pro and just sticking with whatever is on my V2 and got a lulzbot instead. I'm kinda over the ender stuff. Especially the new s1 pro. Paid $500 and it doesn't even support mid print filament changes, along with other things. And the display isn't open source 🤦‍♂️, so now for me to compile a custom firmware, I have to downgrade my screen? 🤦‍♂️🤦‍♂️🤦‍♂️ Seems like 1 step forward, 2 steps back

**Comment by Tapap84** [comment]
Agreed. Now we have no development at all. In glad I didn't clear my
downloads from the last time i got a release. I'd sooner use stock firmware
or my own compilation from main marlin before I use a non jyers/tititopher
branch

**Comment by brunohenriquy** [comment]
Ohh, what happened? Is it still about GPL License and mriscoc?

**Comment by Jonny11bonk** [comment]
> I have a download from 6/18/22 if anyone needs i can email it

What even happened here? I came here to this repo after 6 months and found out so much stuff happening here. Did this turn into the Faithful pack scenario? What happened to titi... you know. I don't even know If I'm allowed to say that here. Why/How did he disappear? I mean, I just download the FW and use it. But, there's something I'm missing, right?

**Comment by Tapap84** [comment]
Not us. I wouldn't trust it unless it's posted by jyers or tititopher

**Comment by Tapap84** [comment]
From my understanding

**Reply by ghost** [reply]
Apparently Miguel thinks that he somehow owns the Marlin distro for the Ender 3, then. huh

Guy doesn't even know how to properly use git.

That's it then. Jyers is done, titito is done and I'm not using Miguel's firmware (too buggy for my use cases).

I'm going to Klipper. Nice while it lasted. I'll make sure my next printer isn't as full of drama as this one on the FW department.

**Comment by Tapap84** [comment]
Join the discord. Turns out tititopher is still making

**Reply by ghost** [reply]
Got an url? I didn’t even know that there was a discord.

**Comment by Tapap84** [comment]
It's in French but Google translate will get you there
https://discord.gg/cjcCCFVj

It's pinned in firmware ender 3

**Reply by ghost** [reply]
Main man! Thank you.

**Reply by anthonylavado** [reply]
@Tapap84 Can you generate another invite? This one is expired.

**Comment by Tapap84** [comment]
https://discord.gg/r4HK5fpJeq

There's a new version that enables linear Advance on tmc 2208 boards
without hardware mods
